lib/tlscontext.c

```diff
@@ -999,6 +1062,21 @@ tls_context_set_key_file(TLSContext *self, const gchar *key_file)
   SSL_CTX_set_default_passwd_cb_userdata(self->ssl_ctx, self->key_file);
 }
 
+gboolean
+tls_context_set_keylog_file(TLSContext *self, gchar *keylog_file_path)
+{
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+  g_free(self->keylog_file_path);
+  msg_warning_once("WARNING: TLS keylog file has been set up, it should only be used during debugging sessions, NOT in production environments",
```

Comment:
Could we skip the `NOT in production environments` part ?
Reasoning, I agree this is not a feature that should be turned on all the time. But it must be turned on in production environment if we want to do a debug session there, and the message would yield otherwise.